tasks/main.yml

```diff
@@ -197,6 +197,20 @@
   changed_when: not __realm_join_output.stderr is
     search("Already joined to this domain")
 
+- name: Configure SSSD settings
+  community.general.ini_file:
+    path: /etc/sssd/sssd.conf
+    state: "{{ item.state | default('present') }}"
+    section: "sssd"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+    create: true
+    owner: root
+    group: root
+    mode: u=rw,g=,o=
+  loop: "{{ ad_integration_sssd_settings }}"
+  notify: Handler for ad_integration to restart services
```

Comment:
I think we're going to need two separate handlers - one for `realmd` and one for `sssd` - it looks like there are cases where changes are made to config files that need `realmd` to be restarted but not `sssd`, and vice versa.  Also, are there cases where a change to `sssd.conf` will require a restart of both `realmd` and `sssd`?

Yes, we could just restart both services every time anything changes, but I would strongly prefer not to restart services unnecessarily.

Reply:
Hi,
I can be better to do two handlers. I can look if you want. Should I do this in the same PR as this one ?

Reply:
> Hi, I can be better to do two handlers. I can look if you want. Should I do this in the same PR as this one ?

Yes, please

Reply:
Hi,
I have updated the PR. I have try to squash to have only 1 commit but I didn't make it.